src/share-menu.js

```diff
@@ -46,6 +52,15 @@ ShareMenu.prototype._fetchPermalink = function () {
 };
 
 /**
+ * @extend
+ */
+ShareMenu.prototype.buildEventData = function (ev) {
```

Comment:
Seems like this is overkill. Can remove this whole function and then do this in `handleOptionClick`:
```
$(ev.target).trigger('insights:local', {type: INSIGHTS_VERBS[data.value]});
```